src/__testHelpers__/mountRoute.js

```
@@ -31,7 +31,7 @@ export default async function mountRoute(route, { authenticated = true } = {}) {
   const axiosMock = axios.create();
 
   if (authenticated) {
-    await store.dispatch(authenticate('test-username', 'password'));
+    await store.dispatch(authenticate('test-username', 'test-password'));
```

> fix a clash with upstream API call mocking
(your commit message)

what it means and what's the issue here?

---

You saw the the Travis `pr` check was failing, yeah? That was because a previously merged PR changed the `axios` mock response for `/api/v1/authenticate` and I failed at the time to change `mountRoute()` to match. 

So when Travis merged this branch into master, the tests which rely upon `mountRoute()` doing authentication for them failed.